lib/mayaUsd/render/vp2RenderDelegate/proxyRenderDelegate.h

```diff
@@ -75,6 +75,11 @@ enum HdVP2SelectionStatus
     kFullyLead          //!< A Rprim is lead (meaning fully lead for instanced Rprims)
 };
 
+// Forward declaration of UsdPointInstancesPickMode so that it can be added as
+// a private member of ProxyRenderDelegate. The values and usage of this enum
+// are otherwise an implementation detail.
+enum class UsdPointInstancesPickMode;
```

Comment:
We are trying to put doc in the headers instead of cpp files. I think it is probably better to have both definition and doc in the header for this case.

Reply:
Addressed in 33c80b7ec3e77cb693bab4a7c8cab853741a0f16.